packages/qa/pages/StudyUpdatePage.ts

```diff
@@ -3,24 +3,259 @@ import { expect, Locator, Page } from '@playwright/test'
+    this.estimatedReopenDD = page.locator('#day').nth(4)
+    this.estimatedReopenMM = page.locator('#month').nth(4)
+    this.estimatedReopenYYYY = page.locator('#year').nth(4)
+    this.ukRecruitmentTarget = page.locator('#recruitmentTarget')
+    this.furtherInfo = page.locator('#furtherInformation')
+    this.buttonUpdate = page.locator('.govuk-button')
+    this.buttonCancel = page.locator('.govuk-button.govuk-button--secondary')
+    this.requestSupport = page.locator('[data-testid="request-support"]')
   }
 
   //Page Methods
   async goto(studyId: string) {
     await this.page.goto(`studies/${studyId}/edit`)
   }
 
-  async assertOnUpdateStudyPage() {
+  async assertOnUpdateStudyPage(studyId: string) {
     await expect(this.pageTitle).toBeVisible()
+    await expect(this.bannerTitle).toContainText('Update study data')
     await expect(this.pageTitle).toContainText('Update study data')
-    await expect(this.page).toHaveURL(/\/studies\/\d+\/edit/)
+    await expect(this.page).toHaveURL(`studies/${studyId}/edit`)
+  }
+
+  async assertSponsorOrg(studySponsors: any) {
+    const findSelectedObject = (sponsors: any[]) => {
+      const cro = sponsors.find((sponsor) => sponsor.OrganisationRole === 'Contract Research Organisation')
+      const ctu = sponsors.find((sponsor) => sponsor.OrganisationRole === 'Managing Clinical Trials Unit')
+      const sponsor = sponsors.find((sponsor) => sponsor.OrganisationRole === 'Clinical Research Sponsor ')
+
+      if (cro) {
+        return cro.OrganisationName
+      } else if (ctu) {
+        return ctu.OrganisationName
+      } else if (sponsor) {
+        return sponsor.OrganisationName
+      }
+    }
+
+    const expectedSponsor = findSelectedObject(studySponsors)
+
+    await expect(this.sponsorOrganisation).toBeVisible()
+    await expect(this.sponsorOrganisation).toHaveText(`Study sponsor: ${expectedSponsor}`)
+  }
+
+  async assertStudyTitle(expectedTitle: string) {
+    await expect(this.studyTitle).toBeVisible()
+    await expect(this.studyTitle).toHaveText(`Study short title: ${expectedTitle}`)
+  }
+
+  async assertGuidanceText(studyType: string) {
+    await expect(this.guidanceText).toBeVisible()
+    if ((studyType = 'Commercial')) {
+      await expect(this.guidanceText).toHaveText(
+        'Changes to the study status, the key dates and recruitment targets will be communicated to RDN, where possible, your changes will update the study record automatically in CPMS, other changes might be subject to review by the RDN team.'
+      )
+    } else {
+      await expect(this.guidanceText).toHaveText(
+        'Changes to the study status, the key dates and recruitment targets will be communicated to RDN, where possible, your changes will update the study record automatically in CPMS, other changes might be subject to review by the RDN team.'
+      )
+    }
+  } // TODO: remove condition if no need for alt text based on study type
```

Comment:
Minor - I think we can remove the condition now as the guidance text shouldn't be based on study type

Reply:
Good point, will do!